etc/column/column.conf

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
+# Log file
+log_file = /var/log/column.log
+
+# Log info level
+log_level = DEBUG
+
+[API]
+server = 0.0.0.0
```

Comment:
I wish you included that in your first comment, my impression was that you wanted to not have any default at all :-) I agree, 127.0.0.1 sounds much better!